src/hooks/usePromptConfirmOnNavigate.ts

```diff
@@ -15,6 +20,10 @@ const usePromptConfirmOnNavigate = ({
   when: boolean;
 }) => {
   const { navigator } = useContext(NavigationContext);
+
+  // TODO: these could be gathered to some file with static data
+  const projectSideBarItems = ['#basics', '#status', '#schedule', '#financial', '#responsiblePersons', '#location', '#projectProgram'];
+ 
```

Comment:
If the idea was to catch if the user is trying to navigate out from the project form, would it be better to match the pathname with a regex patter? It could be something like
`const projectBasicsPathName = /^\/project\/.*\/(basics|new)$/`
It could also help in the future, cause if there's new sections added to the project form, this list would have to be updated, but the regex doesn't.